float64 relative_humidity
float64 variance
